src/reducers/authReducer.js

```diff
@@ -0,0 +1,26 @@
+import { AUTH_ACTIONS } from 'actions/action';
+
+const initState = { nickname: '', isAuthenticated: false };
+
+function authReducer(state = initState, action) {
+  switch (action.type) {
+    case AUTH_ACTIONS.LOGIN:
+      return {
+        ...state,
+        nickname: action.nickname,
+        isAuthenticated: true,
+      };
+
+    case AUTH_ACTIONS.LOGOUT:
+      return {
+        ...state,
+        nickname: '',
+        isAuthenticated: false,
+      };
```

Comment:
이미 merge가 돼서, `새로고침 시 로그인 유지 불가` 에러에 대해 수정한 코드를 코멘트로 적어두겠습니다.
```suggestion
const initState = { isLoading: true, nickname: '', isAuthenticated: false };

function authReducer(state = initState, action) {
  switch (action.type) {
    case AUTH_ACTIONS.LOGIN:
      return {
        ...state,
        isLoading: false,
        nickname: action.nickname,
        isAuthenticated: true,
      };

    case AUTH_ACTIONS.LOGOUT:
      return {
        ...state,
        isLoading: false,
        nickname: '',
        isAuthenticated: false,“
      };
```